tfx/extensions/experimental/kfp_compatibility/kfp_container_component.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions for creating container components from kubeflow components."""
+
+from __future__ import absolute_import
```

Comment:
Does this module need to be Py2 compatible? If not then we can drop this __future__ import